packages/task/src/browser/task-service.ts

```diff
@@ -313,7 +313,7 @@ export class TaskService implements TaskConfigurationClient {
             TERMINAL_WIDGET_FACTORY_ID,
             <TerminalWidgetFactoryOptions>{
                 created: new Date().toString(),
-                id: 'task-' + taskId,
+                id: 'task-' + terminalId,
```

Comment:
I noticed a bug regarding the current terminal id for tasks, as it was impossible to later query all terminals to get the correct id using the `taskId` as the terminal id. Instead, I use the `terminalId` which is unique and does not cause issues later when using the `terminalService` to get the terminal by id. If there is an issue with this implementation please let me know.

Reply:
would it be ok to prefix the id with `terminal` instead of `task` ?